app/assets/stylesheets/styles.scss

```diff
@@ -27,3 +32,24 @@ h1 {
   line-height: 48pt;
   text-align: right;
 }
+
+#key {
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
+#junk {
+  color: $junk_color;
+}
+
+#heirloom {
+  color: $heirloom_color;
+}
+
+#antiquity {
+  color: $antiquity_color;
```

Comment:
Name of variable `antiquity_color` should be written in all lowercase letters with hyphens instead of underscores